### DISCUSSION TITLE
Can we get the GitHub learning lab back?

### DISCUSSION THREAD

**Discussion by ZhaoKunLong** [community/community#31559] [Other Feature Feedback, Questions, & Ideas] [answered]
I have always known that it will be closed today, but I think your guys will definitely come up with better alternatives to it. 
But after I tried GitHub skills, my opinion changed. GitHub skills are really terrible. It does not upgrade any of the original functions of the GitHub learning lab but reduces many useful and friendly functions. It looks like a group of college students just made it.

It is a serious mistake to shut it down now. This is my feedback after the trial.

1. No learning progress bar
2. The course is not grouped as a good sort
3. The course is not gradual
4. Too few courses
6. There is no interaction in the course.

It's like just putting a few repos and then leaving it alone.

Finally, is there any way to get the learning lab back, please? 
Because GitHub skills are really not ready to be the next generation product.

**Comment by allthedoll** [comment, marked as answer]
Most of the stuff from the Lab was moved into [MS Learn](https://docs.microsoft.com/en-us/learn/github/), and that's where the team that was making content puts all of their new stuff now. :)
